library/Customizer/Sections/Search.php

```diff
@@ -28,5 +28,33 @@ public function __construct(string $sectionID)
             ]
             ]
         )));
+
+        
+        Kirki::add_field(\Municipio\Customizer::KIRKI_CONFIG, [
+            'type'        => 'select',
+            'settings'    => 'hero_search_position',
+            'label'       => esc_html__('Hero search position', 'municipio'),
+            'section'     => $sectionID,
+            'default'     => 'centered',
+            'priority'    => 10,
+            'choices'     => [
+                'centered' => esc_html__('Centered', 'municipio'),
+                'bottom' => esc_html__('Bottom', 'municipio'),
+                'top' => esc_html__('Top', 'municipio'),
```

Comment:
Quick fix to ease the process of selecting and avoiding confustion by making the actual order the same as the order of reading. Top to bottom.
```suggestion
                'top' => esc_html__('Top', 'municipio'),
                'centered' => esc_html__('Centered', 'municipio'),
                'bottom' => esc_html__('Bottom', 'municipio'),

```